src/config/risingwave.toml

```diff
@@ -4,7 +4,8 @@ heartbeat_interval_ms = 1000
 [batch]
 
 [streaming]
-checkpoint_interval_ms = 100
+checkpoint_interval_ms = 250
+in_flight_barrier_nums = 10
```

Comment:
10 looks too small for me. Given that generally we have barrier latency of 5s-10s, we should set it to 10s / 250ms = 40.

Reply:
Or there's any other reason for setting this value?

Reply:
I randomly chose this number